resin/auth.py

```diff
@@ -119,6 +148,12 @@ def get_token(self):
         Raises:
             InvalidOption: if not logged in and there is no token in Settings.
 
+        Examples:
+            # If you are logged in.
```

Comment:
Same here (plus another one or two more similar cases I think I saw while scanning the output earlier)